src/content/self-hosted/administration/configuration.mdx

```diff
@@ -633,7 +633,9 @@ There is only one exception where this storage class is overwritten. In case of
 
 ### globalClusterRole
 
-Okteto assings this cluster role to every user via a cluster role binding. By default, this behavior is disabled.
+Okteto assings this cluster role to every user via a cluster role binding.
+The cluster role binding is created for the user service account in the `okteto` namespace, and the `default` service account on every user namespace.
```

Comment:
I don't understand the part about 'cluster role binding is created'. Do we create it, or do we expect the admin to create it?
```suggestion
The cluster role binding is created for the user service account in the `okteto` namespace, and the `default` service account on every user namespace.
```

Reply:
@rberrelleza the cluster role is created by the okteto admin. the cluster role binding is created by okteto to map the cluster role to each okteto account